server/src/main/java/org/cloudfoundry/identity/uaa/audit/JdbcFailedLoginCountingAuditService.java

```diff
@@ -52,6 +52,16 @@ public void log(AuditEvent auditEvent) {
                                                 - saveDataPeriodMillis));
                 super.log(auditEvent);
                 break;
+            case ClientAuthenticationSuccess:
```

Comment:
The preferred method is all tests in the first commit - tests go red. This would limit the number of changes we have to look through.

The second commit, would make all tests go green. We should then see an increase in test coverage, rather than a drop.